addons/account/views/account_move_views.xml

```diff
@@ -94,6 +94,31 @@
                 <kanban class="o_kanban_mobile" create="false" group_create="false">
                     <field name="company_currency_id"/>
                     <field name="partner_id"/>
+                    <field name="payment_id"/>
+                    <field name="product_id"/>
+                    <field name="quantity"/>
+                    <field name="product_uom_category_id"/>
+                    <field name="product_uom_id"/>
+                    <field name="price_unit"/>
+                    <field name="discount"/>
+                    <field name="price_subtotal"/>
+                    <field name="price_total"/>
+                    <field name="sequence"/>
+                    <field name="move_name"/>
+                    <field name="date"/>
+                    <field name="group_tax_id"/>
+                    <field name="tax_line_id"/>
+                    <field name="tax_group_id"/>
+                    <field name="tax_repartition_line_id"/>
+                    <field name="tax_base_amount"/>
+                    <field name="company_id"/>
+                    <field name="recompute_tax_line"/>
+                    <field name="display_type"/>
+                    <field name="is_rounding_line"/>
+                    <field name="exclude_from_invoice_tab"/>
+                    <field name="account_internal_type"/>
+                    <field name="account_internal_group"/>
+
```

Comment:
Some of these are not used in a kanban view, do you still need them here?
payment_id, discount, sequence, move_name, tax_group_id
```suggestion
                    <field name="product_id"/>
                    <field name="quantity"/>
                    <field name="product_uom_category_id"/>
                    <field name="product_uom_id"/>
                    <field name="price_unit"/>
                    <field name="price_subtotal"/>
                    <field name="price_total"/>
                    <field name="date"/>
                    <field name="group_tax_id"/>
                    <field name="tax_line_id"/>
                    <field name="tax_repartition_line_id"/>
                    <field name="tax_base_amount"/>
                    <field name="company_id"/>
                    <field name="recompute_tax_line"/>
                    <field name="display_type"/>
                    <field name="is_rounding_line"/>
                    <field name="exclude_from_invoice_tab"/>
                    <field name="account_internal_type"/>
                    <field name="account_internal_group"/>

```